README.md

```diff
@@ -32,17 +32,17 @@ $ ember exam --server
 
 The idea is that you should be able to replace `ember test` with `ember exam` and never look back.
 
-However, to get the unique features of Ember Exam (described in-depth below), you will need to import functions in `addon-test-support/load.js` to load Ember Exam, to load test manually and to set up qunit callbacks to enable `load-balance` option before your test suite runs. This is done by invoking the `loadEmberExam`, `loadTests`, and `setupQunitCallbacks` function, usually from within `test-helper.js` but it can be done anywhere that executes prior to your tests loading.
+However, to get the unique features of Ember Exam (described in-depth below), you will need to import the `emberExam` object from `addon-test-support/load.js` then call the three functions shown below, usually from within `test-helper.js` but it can be done anywhere that executes prior to your tests loading.
 
 ```js
 // test-helper.js
 import emberExam from 'ember-exam/test-support/load';
 
-emberExam.loadEmberExam();
-emberExam.loadTests();
-emberExam.setupQUnitCallbacks(QUnit);
+emberExam.loadEmberExam(); // To set an instance of testLoader and pass it to testem
+emberExam.loadTests(); // To load tests
+emberExam.setupQUnitCallbacks(QUnit); // To registers QUnit callbacks neeeded for the load-balance option.
```

Comment:
The comment should be 
  To register QUnit callbacks needed for the load-balance option.